client/data/site-migration/landing/logger.ts

```diff
@@ -0,0 +1,38 @@
+import config from '@automattic/calypso-config';
+import debug from 'debug';
+import { logToLogstash } from 'calypso/lib/logstash';
+import type { SiteId } from 'calypso/types';
+
+const debugLog = debug( 'calypso:data:site-migration' );
+
+function safeLogToLogstash( params: Parameters< typeof logToLogstash >[ 0 ] ): void {
+	if ( process.env.NODE_ENV === 'test' ) {
+		return;
+	}
+
+	try {
+		logToLogstash( params );
+	} catch ( error ) {
+		// Fail silently
+	}
+}
```

Comment:
If exceptions are a thing we should worry about in this case, I think we should add the `safeLogToLogstash` to the logstash lib so others can take advantage of this.

Reply:
I totally agree; I would just like to test this approach first before moving to a more public folder.

Reply:
+1 to both your comments